sites/diverseeducation.com/config/navigation.js

```diff
@@ -36,6 +36,7 @@ const awards = [
   { href: '/awards-honors/graduate-scholars/2022', label: 'Graduate Scholars' },
   { href: '/awards-honors/mppwcc', label: 'Most Promising Places to Work: Community Colleges' },
   { href: '/awards-honors/mppwsa', label: 'Most Promising Places to Work: Student Affairs' },
+  // { href: '/awards-honors/top-women', label: 'Top Women' },
```

Comment:
Is this intentionally commented out,? If so can probably just not include it until we need to add it.

Reply:
@Shinsina it was intentional since the Marcos is currently adding content and previous content as well.